Editor/CourseValidation/Context/MemberInfoContext.cs

```diff
@@ -12,11 +13,14 @@ public class MemberInfoContext : IContext
         /// <inheritdoc/>
         public IContext Parent { get; }
 
+        public IData ParentData { get; }
+
         internal MemberInfo MemberInfo { get; }
```

Comment:
Should this be internal? Public? Is it even used outside this class?

Reply:
yes it is